src/k-provider/ovp/loaders/session-loader.js

```diff
@@ -34,7 +34,7 @@ export default class OVPSessionLoader implements ILoader {
    */
   constructor(params: Object) {
     this.requests = this.buildRequests(params);
-    this._partnerId = params.partnerId;
+    this._widgetId = params.widgetId;
```

Comment:
If widgetId is not set then just populate it with _partnerId here and then you don’t need to reference the widgetId||’_’+partnerId all over ovp provider

Reply:
I mean that if the params contain only partnerId then populate the `_widgetId` param yourself.
```javascript
constructor(params: Object) {
    ...
    this._partnerId = params.partnerId;
    this._widgetId = params.widgetId || '_' + params.partnerId;
```

so you can then pass down the `widgetId` to the `OVPSessionLoader`

Reply:
so, what is the different between my code and your suggestion?